src/main.rs

```diff
@@ -32,6 +32,7 @@ fn run_blocking() -> anyhow::Result<()> {
         .on_thread_stop(|| {
             TRACING_GUARD.take();
         })
+        .enable_all()
```

Comment:
Why are we enabling all here?

Reply:
duplicate as https://github.com/tailcallhq/tailcall/pull/1647#discussion_r1555022010 marking this as resolved